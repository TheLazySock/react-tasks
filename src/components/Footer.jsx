import React from 'react';

const Footer = () => (
    <footer className="footer">
        <label>netflixroulette</label>
    </footer>
);

export default Footer;
