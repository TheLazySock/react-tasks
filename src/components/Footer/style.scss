@import '../../assets/constants.scss';

.footer {
    width: 100%;
    height: 35px;
    line-height: 30px;
    display: block;
    background-color: #211f1f;
    color: $red;
    position: relative;
    flex-grow: 0;
    & > label {        
        vertical-align: middle;
        margin-left: 15px;
    }
}