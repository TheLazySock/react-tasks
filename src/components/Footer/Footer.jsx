import React from 'react';

import style from './style.scss';

const Footer = () => (
    <footer className={style.footer}>
        <label>netflixroulette</label>
    </footer>
);

export default Footer;
