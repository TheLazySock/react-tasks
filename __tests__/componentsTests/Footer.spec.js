import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../src/components/Footer';
import { configure } from 'enzyme';

describe('>>>FOOTER --- snapshot', () => {
    it('Footer have to rendered succesfully', () => {
        const renderedFooter = renderer.create(<Footer />).toJSON();
        expect(renderedFooter).toMatchSnapshot();
    });
});