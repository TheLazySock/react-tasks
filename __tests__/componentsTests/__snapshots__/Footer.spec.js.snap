// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>FOOTER --- snapshot Footer have to rendered succesfully 1`] = `
<footer
  className="footer"
>
  <label>
    netflixroulette
  </label>
</footer>
`;
